config/crd/bases/azconfig.io_azureappconfigurationproviders.yaml

```diff
@@ -345,9 +347,3 @@ spec:
     storage: true
     subresources:
       status: {}
-status:
```

Comment:
I don't expect to remove this section

Reply:
It's auto-generated by make command. What's the influence of removing this section?